content/sensu-go/5.18/getting-started/enterprise.md

```diff
@@ -11,25 +11,33 @@ menu:
 ---
 
 Sensu Go offers commercial features designed for monitoring at scale.
-[Contact the Sensu sales team][1] for a personalized demo and free trial.
-All commercial features are available for free in the packaged Sensu Go distribution up to an entity limit of 100.
-See the [announcement on our blog][7] for more information.
+All commercial features are available for free in the packaged Sensu Go distribution [up to an entity limit of 100][7].
+
+If you have more than 100 entities, [contact the Sensu sales team][1] for a personalized demo and free trial.
```

Comment:
```suggestion
If you have more than 100 entities, [contact the Sensu sales team][1] for a free trial.
```
As one of our current OKRs is to reduce custom demos, I think we should remove that bit.